packages/notte-core/src/notte_core/browser/snapshot.py

```
@@ -40,7 +40,7 @@ class SnapshotMetadata(BaseModel):
     url: str
     viewport: ViewportData
     tabs: list[TabsData]
-    timestamp: dt.datetime = field(default_factory=dt.datetime.now)
+    timestamp: dt.datetime = field(default_factory=lambda: dt.datetime.now())
```

Pylint (3.3.7)

[error] Invalid usage of field(), it should be used within a dataclass or the make_dataclass() function.

(E3701)